python/graphstorm/utils.py

```diff
@@ -365,7 +392,7 @@ def save_profile(self):
             profile_path = os.path.join(self._profile_path, f"{self._rank}.csv")
             data_frame = pd.DataFrame(runtime)
             data_frame.to_csv(profile_path, float_format='%.3f', index=False)
-            print(f"save profiling in {profile_path}")
+            logging.debug("save profiling in %s", profile_path)
```

Comment:
Let's make it logging.info, so users can directly know where the profile data are stored.